pkg/server/grpc/tracee_test.go

```diff
@@ -124,11 +124,11 @@ func Test_convertEventWithThreat(t *testing.T) {
 		},
 	}
 
-	protoEvent := convertTraceeEventToProto(traceEvent)
+	protoEvent, err := convertTraceeEventToProto(traceEvent)
+	assert.NoError(t, err)
 
 	assert.Equal(t, "An attempt to abuse the Docker UNIX ..", protoEvent.Threat.Description)
-	assert.Equal(t, "privilege-escalation", protoEvent.Threat.MitreTactic.Name)
-	assert.Equal(t, "Exploitation for Privilege Escalation", protoEvent.Threat.MitreTechnique.Name)
-	assert.Equal(t, "attack-pattern--b21c3b2d", protoEvent.Threat.MitreTechnique.Id)
-	assert.Equal(t, "T1068", protoEvent.Threat.MitreTechnique.ExternalId)
+	assert.Equal(t, "privilege-escalation", protoEvent.Threat.Mitre.Tactic.Name)
+	assert.Equal(t, "Exploitation for Privilege Escalation", protoEvent.Threat.Mitre.Technique.Name)
+	assert.Equal(t, "attack-pattern--b21c3b2d", protoEvent.Threat.Mitre.Technique.Id)
```

Comment:
Regarding my previous comment - the Id should be the T1068 and not the attack-...